alignak/acknowledge.py

```diff
@@ -50,49 +50,45 @@
 
 """
 
+from alignak.property import StringProp, IntegerProp
 from alignak.alignakobject import AlignakObject
 
 
-class Acknowledge(AlignakObject):  # pylint: disable=R0903
+class Acknowledge(AlignakObject):
     """
-    Allows you to acknowledge the current problem for the specified service.
+    Allows to acknowledge the current problem for the specified service.
     By acknowledging the current problem, future notifications (for the same
-    servicestate) are disabled.
-    """
+    service state) are disabled.
+
+    If the "sticky" option is set to one (1), the acknowledgement
+    will remain until the service returns to an OK state. Otherwise
+    the acknowledgement will automatically be removed when the
+    service changes state. In this case Web interfaces set a value
+    of (2).
 
-    properties = {
-        'uuid': None,
-        'sticky': None,
-        'notify': None,
-        'end_time': None,
-        'author': None,
-        'comment': None,
-    }
-    # If the "sticky" option is set to one (1), the acknowledgement
-    # will remain until the service returns to an OK state. Otherwise
-    # the acknowledgement will automatically be removed when the
-    # service changes state. In this case Web interfaces set a value
-    # of (2).
-    #
-    # If the "notify" option is set to one (1), a notification will be
-    # sent out to contacts indicating that the current service problem
-    # has been acknowledged.
-    #
-    # If the "persistent" option is set to one (1), the comment
-    # associated with the acknowledgement will survive across restarts
-    # of the Alignak process. If not, the comment will be deleted the
-    # next time Alignak restarts.
+    If the "notify" option is set to one (1), a notification will be
+    sent out to contacts indicating that the current service problem
+    has been acknowledged.
 
-    def serialize(self):
-        """This function serialize into a simple dict object.
-        It is used when transferring data to other daemons over the network (http)
+    If the "persistent" option is set to one (1), the comment
+    associated with the acknowledgement will survive across restarts
+    of the Alignak process. If not, the comment will be deleted the
+    next time Alignak restarts.
 
-        Here we directly return all attributes
+    """
 
-        :return: json representation of a Acknowledge
-        :rtype: dict
-        """
-        return {'uuid': self.uuid, 'ref': self.ref, 'sticky': self.sticky, 'notify': self.notify,
-                'end_time': self.end_time, 'author': self.author, 'comment': self.comment,
-                'persistent': self.persistent
-                }
+    properties = AlignakObject.properties.copy()
+    properties.update({
+        'sticky':
```

Comment:
We could convert the number passed by external command with boolean for `sticky`, `notify`, `persistent`